raspberry_pi_installation.md

```diff
@@ -70,17 +70,10 @@ Now you need to start the Node Server. Type...
 You should see the terminal output "signalk-server running at 0.0.0.0:3000" as shown below...
 
     $ signalk-server --sample-nmea0183-data
-    Using default configuration path: /home/pi/.signalk
-    Settings file does not exist, using empty settings
     Using sample data from /usr/lib/node_modules/signalk-server/samples/plaka.log
-    *** WARNING *** The program 'node' uses the Apple Bonjour compatibility layer of Avahi.
```

Comment:
Have the Avahi warning disappeared? That would be indeed nice, they look pretty scary for somebody starting out.

Reply:
Jepp, did a fresh install for some time ago and now running 1.18 w/o any warnings